src/web/components/ArticleTitle.tsx

```diff
@@ -14,6 +14,7 @@ type Props = {
 	sectionUrl: string;
 	guardianBaseURL: string;
 	badge?: BadgeType;
+	editionId?: Edition;
```

Comment:
```suggestion
	editionId: Edition;
```

Can we make this a required prop? Does that work?